test/physics_test/IntegratorComparisonTest.cpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file IntegratorComparisonTest.cpp
+ * @author F. Gratl
+ * @date 18.02.22
+ */
+
+#include "IntegratorComparisonTest.h"
+#include "satellitePropagator/physics/AccelerationAccumulator.h"
+#include "satellitePropagator/physics/YoshidaIntegrator.h"
+
+TEST(IntegratorComparisonTest, YoshidaVsManually)
```

Comment:
we should definitely add some more tests for this, I reckon (especially given the mentioned timestep dilemma) but we can also create an issue for that and not do it here?

Reply:
-> #9